notebooks/templates/Inspection_jobs_analysis_TEMPLATE.ipynb

```diff
@@ -7,7 +7,7 @@
    },
```

Comment:
<ul><li>The global variables can be in capital cases as they are default parameters of the functions below.</li><li>missing spaces</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thoth-station/notebooks/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>